ansible/roles/media-licenses/tasks/main.yml

```
@@ -0,0 +1,15 @@
+- name: media licenses sync CronJob
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: vidis-media-activation-data-sync-cronjob.yml.j2
+  tags:
+    - cronjob
+
+- name: media licenses sync CronJob ConfigMap
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: vidis-media-activation-data-sync-cronjob-configmap.yml.j2
+  tags:
+    - configmap
```

the configmap shoud be created bevore the cronjob ad missing the with condion to activate it
```suggestion
- name: media licenses sync CronJob ConfigMap
  kubernetes.core.k8s:
    kubeconfig: ~/.kube/config
    namespace: "{{ NAMESPACE }}"
    template: vidis-media-activation-data-sync-cronjob-configmap.yml.j2
  tags:
    - configmap

- name: media licenses sync CronJob
  kubernetes.core.k8s:
    kubeconfig: ~/.kube/config
    namespace: "{{ NAMESPACE }}"
    template: vidis-media-activation-data-sync-cronjob.yml.j2
  tags:
    - cronjob
```